test/base/src/androidTest/kotlin/io/realm/test/shared/ImportTests.kt

```diff
@@ -106,7 +106,7 @@ class ImportTests {
         val clone = realm.writeBlocking { copyToRealm(root) }
 
         assertNotNull(clone)
-        assertEquals(2, realm.objects(Sample::class).count())
+        assertEquals(2L, realm.query(Sample::class).count().find())
```

Comment:
Maybe tested somewhere else, but we should probably have a test for checking the size of the result
```
realm.query(Sample::class).find().size
``` 
too

Reply:
`size` is tested in https://github.com/realm/realm-kotlin/blob/372f861f86a9f5dfbf2c9f2c3134e7ad645093ba/test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt#L88
and
https://github.com/realm/realm-kotlin/blob/372f861f86a9f5dfbf2c9f2c3134e7ad645093ba/test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt#L90